Remotion/Web/Core/UI/Controls/ValidationError.cs

```diff
@@ -144,7 +142,7 @@ public HtmlGenericControl ToLabel (string cssClass)
     {
       HtmlGenericControl label = new HtmlGenericControl ("label");
 
-      label.Controls.Add (ToSpan (ValidationMessage));
+      label.Controls.Add (ToSpan (ValidationMessage.GetValue()));
```

Comment:
Great, I just found a bug, here and with the other usage of ToSpan.

ToSpan gets a cssclass, not the validationmessage. => pass `null`, since we already add the css lcass to the parent element.

This should be a dedicated issue, though.

Reply:
https://re-motion.atlassian.net/browse/RM-8236